Author: Adam Gandelman <adamg@canonical.com>
Date:  Mon Aug 27 21:39:28 PDT 2012
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/horizon/+bug/1037349
Subject: Set login redirects to match default Apache config.

Ubuntu now defaults the horizon's web root to /horizon instead of /.  The auth
views need to be properly configured to land users at the correct place after
auth actions.

Index: horizon-2012.2/openstack_dashboard/local/local_settings.py.example
===================================================================
--- horizon-2012.2.orig/openstack_dashboard/local/local_settings.py.example	2012-08-27 21:35:36.079291684 -0700
+++ horizon-2012.2/openstack_dashboard/local/local_settings.py.example	2012-08-27 21:36:31.295289656 -0700
@@ -142,3 +142,8 @@
             }
         }
 }
+
+# Default Ubuntu apache configuration uses /horizon as the application root.
+# Configure auth redirects here accordingly.
+LOGIN_URL='/horizon/auth/login/'
+LOGIN_REDIRECT_URL='/horizon'
